lighthouse-core/lib/traces/pwmetrics-events.js

```diff
@@ -18,6 +18,22 @@
 
 const log = require('../../../lighthouse-core/lib/log.js');
 
+/**
+ * @param {!Object} object
+ * @param {string} path
+ * @return {*}
+ */
+function safeGet(object, path) {
```

Comment:
not wild about this, mostly because it completely disables typing. Is the issue that the intermediate objects are sometimes undefined?